packages/ppx_sexp/ppx_sexp.0.2.2/opam

```diff
@@ -0,0 +1,15 @@
+opam-version: "1.2"
+maintainer: "Jonathan Chan <jyc@fastmail.fm>"
+authors: "Jonathan Chan <jyc@fastmail.fm>"
+homepage: "https://bitbucket.org/jyc/ppx_sexp"
+bug-reports:
+  "https://bitbucket.org/jyc/ppx_sexp/issues?status=new&status=open"
+license: "BSD 2-Clause"
+dev-repo: "https://bitbucket.org/jyc/ppx_sexp/src"
```

Comment:
Could you please change the dev-repo URL to
`hg://https://bitbucket.org/jyc/ppx_sexp`?